docs/docker/example.md

```diff
@@ -0,0 +1,132 @@
+# Example of creating a development environment with Docker
+
+This project come with a set of small projects that are bundled together by using Docker.
+
+## Dockerfile
+
+Using an image in a raw way can be useful for quick prototyping. However sometimes when you need to do more things internally e.g installing other packages or running command that need to be done before launching your project. a Dockerfile allow you to add more process overhead on top of a based image.
```

Comment:
Using a raw image can be useful for quick prototyping. However when you need to make internal work (e.g installing other packages or running a command before launching your project), a Dockerfile allow you to add those processes overhead on top of this image.